src/nusoap.php

```diff
@@ -908,11 +908,15 @@ function getmicrotime()
     function varDump($data)
     {
         ob_start();
-        var_dump($data);
+        var_dump($this->sanitize($data));
         $ret_val = ob_get_contents();
         ob_end_clean();
         return $ret_val;
     }
+    
+    function sanitize($value) {
+        return htmlspecialchars(strip_tags($value), ENT_COMPAT, 'utf-8');
```

Comment:
Running both `strip_tags` and `htmlspecialchars` doesn't make much sense. In general, trying to write an all-purpose "sanitize" method is a bad idea: escaping should be related to the context where you're outputting or using something.